tests/bettertransformer/test_bettertransformer_encoder.py

```diff
@@ -80,37 +79,12 @@ def prepare_inputs_for_class(self, model_id=None):
         }
         return input_dict
 
-    def _loop_all_classes(self):
-        r"""
-        An utility function to automatically loop over all classes to test.
-        this is a generator that will generate random config class for each
-        model to test.
-        """
-        for layer_class in BETTER_TRANFORMER_LAYERS_MAPPING_DICT.keys():
-            if layer_class == "EncoderLayer":
-                # Hardcode it for FSMT - see https://github.com/huggingface/optimum/pull/494
-                class_name = "FSMT"
-            elif layer_class == "TransformerBlock":
-                # Hardcode it for distilbert - see https://github.com/huggingface/transformers/pull/19966
-                class_name = "DistilBert"
-            elif "EncoderLayer" in layer_class:
-                class_name = layer_class[:-12]
-            else:
-                class_name = layer_class[:-5]
-            random_config = getattr(transformers, class_name + "Config")
-            yield random_config()
-
     def test_dict_class_consistency(self):
-        r"""
-        A test to check if the modified dictionnary is consistent (same number of keys + successfully import
-        the correct `PreTrainedModel` module). This also checks if the list of models to test has the same
-        number of classes than the list of convertable models.
         """
-        for keys in BETTER_TRANFORMER_LAYERS_MAPPING_DICT.keys():
-            self.assertTrue(("Layer" in keys) or ("Block" in keys))
-
-        ALL_SUPPORTED_HF_CLASSES = convert_to_hf_classes(BETTER_TRANFORMER_LAYERS_MAPPING_DICT)
-        self.assertEqual(len(ALL_SUPPORTED_HF_CLASSES.keys()), len(BETTER_TRANFORMER_LAYERS_MAPPING_DICT.keys()))
```

Comment:
In my opinion this test is always passing so I removed it. Let me know if I misunderstand it.

Reply:
Correct, I see also that you have added a new test